pkg/services/ngalert/provisioning/notification_policies.go

```diff
@@ -25,47 +25,39 @@ func NewNotificationPolicyService(am AMConfigStore, prov ProvisioningStore, xact
 	}
 }
 
-// TODO: move to Swagger codegen
-type EmbeddedRoutingTree struct {
```

Comment:
Drop this inlined thing entirely. Move to `apimodels.Route` which now has all the fields we need.